apps/core/tests/test_previous_page_and_next_page.py

```diff
@@ -0,0 +1,32 @@
+from django.test import TestCase
+from wagtail.core.models import Page
+
+from apps.core.factories import HomePageFactory
+
+from ..templatetags.core_tags import next_page, previous_page
+
+
+class TestNavigation(TestCase):
+    def setUp(self):
+        self.home = HomePageFactory()
+        self.a = Page(title="a", slug="a", show_in_menus=True)
+        self.home.add_child(instance=self.a)
+        self.ab = Page(title="ab", slug="ab", show_in_menus=True)
+        self.a.add_child(instance=self.ab)
+        self.ac = Page(title="ac", slug="ac", show_in_menus=True)
+        self.a.add_child(instance=self.ac)
+        self.b = Page(title="b", slug="b", show_in_menus=True)
+        self.home.add_child(instance=self.b)
+
+    def test_previous_page_filter(self):
+        self.assertEqual(previous_page(self.a).specific, self.home)
+        self.assertEqual(previous_page(self.ab), self.a)
```

Comment:
Isn’t this the same test as the one before?

Reply:
Are you talking about `test_previous_page_filter` and `test_next_page_filter`? I don't understand which test are you talking about.

Reply:
Ohh okay I got it. I added those tests so that it covers every type of page (home and content). Yeah the logic is same for both of them and we can remove one if we want.